db/user.py

```diff
@@ -44,6 +44,8 @@ def get_by_token(token):
         return dict(row) if row else None
 
 
+
```

Comment:
I do use `-p` after you suggested this a few weeks earlier. I also review twice before the commits. This code change was pushed 15 days ago and i did not touch the line recently. Its also that sometimes such things just slip out of sight. I'll try not to push them again. :)